app/javascript/dashboard/routes/dashboard/settings/auditlogs/Index.vue

```diff
@@ -1,137 +1,126 @@
-<script>
-import { mapGetters } from 'vuex';
+<script setup>
 import { useAlert } from 'dashboard/composables';
 import { messageTimestamp } from 'shared/helpers/timeHelper';
+import { useStoreGetters, useStore } from 'dashboard/composables/store';
 import TableFooter from 'dashboard/components/widgets/TableFooter.vue';
+import BaseSettingsHeader from '../components/BaseSettingsHeader.vue';
 import {
   generateTranslationPayload,
   generateLogActionKey,
 } from 'dashboard/helper/auditlogHelper';
+import { computed, onMounted, watch } from 'vue';
+import { useI18n } from 'dashboard/composables/useI18n';
+import { useRoute } from 'dashboard/composables/route';
+import router from '../../..';
```

Comment:
@pranavrajs Can we use `useRouter` from `composables/route`?

Reply:
Done.